bc/core/utils/microservices.py

```diff
@@ -0,0 +1,33 @@
+from io import BytesIO
+from typing import IO
+from zipfile import ZipFile
+
+import requests
+from django.conf import settings
+
+
+def get_thumbnails_from_range(
+    document: bytes, page_range: str
+) -> list[IO[bytes]]:
+    """
+    Returns a list that contains a thumbnail(as a binary object) for each
+    page requested.
+
+    Args:
+        document (bytes): document content as bytes
+        page_range (str): str representation of the list of pages requested
+
+    Returns:
+        list[bytes]: list of thumbnails
+    """
+    thumbnails = requests.post(
+        f"{settings.DOCTOR_HOST}/convert/pdf/thumbnails/",
+        data={"pages": page_range, "max_dimension": "1920"},
+        files={"file": (f"dummy.pdf", document)},
+        timeout=4 * 60,
```

Comment:
This is very long to do some simple PDF work. I would expect this to complete in less than a minute? If we have retries anyway, it's probably better to make this a minute and if that's not enough, let the retry take care of it.

Reply:
understood!. I will tweak the timeout.